Microsoft.Toolkit.Uwp.UI/Brushes/BackdropBlurBrush.cs

```diff
@@ -0,0 +1,108 @@
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+// Example brush from https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.xamlcompositionbrushbase
+ 
+using Microsoft.Graphics.Canvas.Effects;
+using Windows.UI;
+using Windows.UI.Composition;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Brushes
+{
+    /// <summary>
+    /// The <see cref="BackdropBlurBrush"/> is a <see cref="Brush"/> that blurs whatever is behind it in the application.
+    /// </summary>
+    public class BackdropBlurBrush : XamlCompositionBrushBase
+    {
+        /// <summary>
+        /// Identifies the <see cref="Amount"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty AmountProperty = DependencyProperty.Register(
+            nameof(Amount),
+            typeof(double),
+            typeof(BackdropBlurBrush),
+            new PropertyMetadata(0.0, new PropertyChangedCallback(OnAmountChanged)));
+
+        /// <summary>
+        /// Gets or sets the amount of gaussian blur to apply to the background.
+        /// </summary>
+        public double Amount
+        {
+            get { return (double)GetValue(AmountProperty); }
+            set { SetValue(AmountProperty, value); }
+        }
+
+        private static void OnAmountChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var brush = (BackdropBlurBrush)d;
+
+            // Unbox and set a new blur amount if the CompositionBrush exists.
+            brush.CompositionBrush?.Properties.InsertScalar("Blur.BlurAmount", (float)(double)e.NewValue);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BackdropBlurBrush"/> class.
+        /// </summary>
+        public BackdropBlurBrush()
+        {
+            this.FallbackColor = Colors.Transparent;
+        }
+
+        /// <summary>
+        /// Initializes the Composition Brush.
+        /// </summary>
+        protected override void OnConnected()
+        {
+            // Delay creating composition resources until they're required.
+            if (CompositionBrush == null)
+            {
+                // Abort if effects aren't supported.
+                if (!CompositionCapabilities.GetForCurrentView().AreEffectsSupported())
+                {
+                    return;
+                }
+
+                var backdrop = Window.Current.Compositor.CreateBackdropBrush();
+
+                // Use a Win2D blur affect applied to a CompositionBackdropBrush.
+                var graphicsEffect = new GaussianBlurEffect
+                {
+                    Name = "Blur",
+                    BlurAmount = (float)this.Amount,
```

Comment:
**This** should be removed

Reply:
The bluramount should be removed? Why?

Reply:
Why should the blur amount be removed?

Reply:
He's saying using the `this.` prefix should be removed. ;)

Reply:
Ah, now that makes sense!